src/pages/Detail/Styled.Detail.js

```diff
@@ -0,0 +1,213 @@
+import styled from 'styled-components';
+
+const S = {};
+
+S.Container = styled.section`
+  position: relative;
+  padding: 30px 100px 100px;
+`;
+
+// 작성자 정보
+S.Editor = styled.article`
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  min-height: 56px;
+  margin: 0 auto;
+  max-width: 1200px;
+`;
+
+S.ProfileImg = styled.div`
+  width: 56px;
+  height: 56px;
+  border: 1px solid lightgray;
+  border-radius: 50%;
+  overflow: hidden;
+`;
+
+S.Img = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+`;
+
+S.EditorInfo = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  margin-left: 20px;
+`;
+
+S.Id = styled.div`
+  display: flex;
+  gap: 10px;
+  font-size: 15px;
+`;
+
+S.Follow = styled.a`
+  color: #1f901f;
+`;
+
+S.Text = styled.div`
+  font-size: 14px;
+  color: ${props => props.theme.lightGrey};
```

Comment:
함수 형태이기 때문에, 다음과 같이 구조분해할당해서 쓸 수도 있습니다. 참고해주세요!
```suggestion
  color: ${({ theme }) => theme.lightGrey};
```